testdata/serviceconfig.yml

```diff
@@ -7,6 +7,23 @@ rules:
   aspects:
   - kind: quotas
     params:
+  - kind: metrics
+    adapter: "prometheus"
+    params:
+      metrics:
+      - descriptor: "request_count"
+        value: "1"
```

Comment:
YAML mandates integers be quoted if they are used in string fields (e.g. "args")